### PR TITLE
Final pushes to the concurrency presentation 

### PR DESCRIPTION
**Pull request by devarsh**

### CODE REVIEW

File: assets_cdd/code/moreExamples/pubsub.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		pub2: []chan string{sub2, sub3},
+	}
+	go publish(subscriptions)
+	pub1 <- "Hello Golang World!"
+	pub2 <- "Hi Ahmedabad Meetup!"
+	pub1 <- "Goodbye, Cruel World!"
+	//Register a subscriber
+	subscriptions[pub1] = append(subscriptions[pub1], sub3)
+	pub1 <- "Just kidding!"
+	time.Sleep(2 * time.Second)
```
Comment (girishramnani) [inline assets_cdd/code/moreExamples/pubsub.go:56]: Could we use wait group here?. Merging now. Will edit afterwards.

**Review by girishramnani — APPROVED** [review]

File: assets_cdd/code/moreExamples/pubsub.go (hunk 1 above)
Comment (devarsh) [reply assets_cdd/code/moreExamples/pubsub.go:56]: Yes, i'll change that

**Comment by devarsh** [comment]
I'm still not really happy with the ratelimit example, i wanted to use this [library](https://github.com/beefsack/go-rate) which is small and precise, but sadly it wasn't using any channels.

**Comment by girishramnani** [comment]
We can add it as a special mention

**Pull request merged**